src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1614,7 +1614,7 @@ open class StateManagerAdaptor(
                 }
             }
             retrieveMarketHistoricalFundings()
-            retrieveMarketCandles()
+            if (appConfigs.subscribeToCandles) retrieveMarketCandles()
```

Comment:
Should we just move this check inside `retrieveMarketCandles()`? (can rename to `maybeRetrieveMarketCandles()` for clarity) Would prevent someone from accidentally adding a call somewhere else without the check.

Reply:
yee! i think we can also align with whether to adopt this pattern for the other functions for v2 as well -- but will leave that for now and keep this small